rust/arrow/src/ipc/convert.rs

```diff
@@ -397,20 +409,24 @@ mod tests {
                 Field::new("time32[ms]", DataType::Time32(TimeUnit::Millisecond), false),
                 Field::new("time64[us]", DataType::Time64(TimeUnit::Microsecond), false),
                 Field::new("time64[ns]", DataType::Time64(TimeUnit::Nanosecond), true),
-                Field::new("timestamp[s]", DataType::Timestamp(TimeUnit::Second), false),
+                Field::new(
```

Comment:
I'll amend some of these to test timezone roundtrips, only noticed now that I haven't covered them.

Reply:
Done